doc/rtd/explanation/kernel-cmdline.rst

```diff
@@ -62,8 +87,9 @@ Then, have the following content at that url:
    positives, ``cloud-init`` requires the content to start with
    ``#cloud-config`` for it to be considered.
 
+
 .. note::
 
-   The ``cloud-config-url=`` is un-authed http GET, and contains credentials.
```

Comment:
Requesting a security review of this change from here to the end of this file, please.

I propose a statement that should hopefully last the test of time a bit better than the current docs.

Reply:
could an encrypted alternative to `cloud-config-url=` be suggested here?

instead of "un-authed", I would say "unencrypted"

*if* credentials are needed, the channel is unlikely to be trusted. perhaps a strongly worded id est could be added to the last sentence, like: `... via trusted channels (i.e., within a closed system).`

Reply:
> instead of "un-authed", I would say "unencrypted"

+1 thanks

> could an encrypted alternative to cloud-config-url= be suggested here?

Unfortunately, this feature doesn't have an encrypted alternative to suggest.

> ... via trusted channels (i.e., within a closed system).

Thanks for the suggestions. I've added them in the latest commit.